packages/multicast_dns/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: multicast_dns
+description: Dart package for mDNS queries (e.g. Bonjour, Avahi).
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/packages/tree/master/packages/mdns
+version: 0.1.0
+
+dependencies:
+  args: ^1.5.0
```

Comment:
since you only depend on this for demos, it's probably best to move the demos into a subdirectory with their own pubspec.yaml that doesn't affect the clients of the package.

Reply:
I'm not sure of the best way to do that - the things asking for this are in the `bin/` directory, and AFAIK if I put a pubspec in there you can no longer `pub global activate` this to use those tools. 

Is there some way to have it both ways?  If not, I suppose it's not a big deal to lose those tools.

Reply:
For now I'll just move them to `example/` and put a pubspec in there.  It's not super-important that this particular package be accessible that way, a separate one could be published with more features anyway.

Reply:
The tools say they're just there as demos. If the tools are actually intended to be useful that's a different matter. I don't think they are, though?